src/api/app/views/webui2/webui/search/index.html.haml

```diff
@@ -1,6 +1,47 @@
 - @pagetitle = 'Search'
+= content_for :ready_function do
+  :plain
+    $('#search-text').focus();
 
 .card.mb-3
   = render partial: 'tabs'
   .card-body
     %h3 Search for packages or projects:
+    .form-row.justify-content-center
```

Comment:
`.form-row` should be used inside a `<form>` tag. Replacing it by `d-flex` also works, so I would suggest doing so.

Reply:
Much better, thanks!